client.go

```diff
@@ -338,12 +338,16 @@ func (c *TunaSessionClient) getPubAddrsFromRemote(ctx context.Context, remoteAdd
 	if err != nil {
 		return nil, err
 	}
+	if pubAddrs.SessionClosed {
+		return pubAddrs, ErrClosed
```

Comment:
I think we should return `nil, ErrClosed`

Reply:
Yes, modified